File: nodejs-microservices-tutorial/src/application/services/UserEventHandler.ts
export class UserEventHandler {
  async handleUserCreated(event: any): Promise<void> {
    console.log('📧 Sending welcome email to:', event.email);
    console.log('📱 Sending SMS notification to:', event.email);
    // In a real app, you'd call email/SMS services here
  }
}

File: fastapi-microservices-tutorial/app/application/services/user_event_handler.py
class UserEventHandler:
    async def handle_user_created(self, event: dict) -> None:
        """Handle user created event"""
        print(f"📧 Sending welcome email to: {event['email']}")
        print(f"📱 Sending SMS notification to: {event['email']}")
        # In a real app, you'd call email/SMS services here
